docs/nodes/reliability/metrics.md

```diff
@@ -59,13 +92,63 @@ The metrics for a CRN have the following form
     "as_name":"INTERNET-SERVICE-PROVIDER, AD",
     "base_latency":0.9623174667358398,
     "diagnostic_vm_latency":0.06729602813720703,
-    "full_check_latency":0.5257446765899658
+    "full_check_latency":0.5257446765899658,
+    "diagnostic_vm_ping_latency": 0.148196"
 }
 ```
 
-## Publishing
+## Analyzing
+
+The [scores](../../nodes/reliability/scores.md) are computed based on the metrics, in a reproducible manner.
+
+Metrics messages can be found:
+
+### On the Message Explorer
+
+[https://explorer.aleph.im/messages?showAdvancedFilters=1&channels=aleph-scoring&sender=0x4D52380D3191274a04846c89c069E6C3F2Ed94e4](https://explorer.aleph.im/messages?showAdvancedFilters=1&channels=aleph-scoring&sender=0x4D52380D3191274a04846c89c069E6C3F2Ed94e4)
+
+### Using the Python SDK
+
```

Comment:
Could maybe here add the link to the aleph-sdk-python doc ?

Reply:
Fixed